browser/resources/settings/brave_default_extensions_page/brave_default_extensions_browser_proxy.js

```diff
@@ -19,6 +19,8 @@ cr.define('settings', function () {
     setTorEnabled (value) {}
     isTorEnabled () {}
     isTorManaged () {}
+    setWidevineEnabled() {}
```

Comment:
I vaguely remember that there is a common way for adding trivial pref toggles, without much boilerplate. Maybe @petemill could help

Reply:
Profile prefs can be toggled very simply w/o this kind of code.
but I don't know we can do same way for local prefs.

Reply:
gotcha